build.sbt

```diff
@@ -15,93 +15,20 @@ lazy val sslConfigCore = project.in(file("ssl-config-core"))
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.NotEqual$"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.Equal$"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.OpSym$NE$"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.MoreThanOrEqual"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.NotEqual"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.Ciphers$"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.AlgorithmConstraint$"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.AlgorithmChecker"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.MoreThan$"),
-      ProblemFilters.exclude[MissingClassProblem]("com.typesafe.sslconfig.ssl.OpSym$LT$"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$7"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$8"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$9"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$10"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$11"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$12"),
-      ProblemFilters.exclude[IncompatibleResultTypeProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$13"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$14"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.SSLConfigSettings.<init>$default$15"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.SSLLooseConfig.<init>$default$6"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.SSLLooseConfig.<init>$default$7"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.ConfigSSLContextBuilder.buildCompositeKeyManager"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.ConfigSSLContextBuilder.buildCompositeTrustManager"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.ConfigSSLContextBuilder.buildKeyManager"),
-      ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.ssl.ConfigSSLContextBuilder.buildTrustManager")
-    ),
-    libraryDependencies += Library.parserCombinators(scalaVersion.value),
```

Comment:
Surprising, but seems OK to me. Downstream projects that needed parser combinators but didn't depend on them directly can start depending on them directly.